lib/ex_uid2/dsp.ex

```diff
@@ -0,0 +1,77 @@
+defmodule ExUid2.Dsp do
+  alias ExUid2.Encryption
+  alias ExUid2.Keyring
+  alias ExUid2.Api
+  use GenServer
+
+  @table_name __MODULE__
+  @keyring :uid2_decryption_keyring
+  @refresh_rate_ms 5000
+
+  require Logger
+
+  def start_link(opts \\ []) do
+    :ets.new(@table_name, [:named_table, {:read_concurrency, true}, :public])
```

Comment:
Is there a particular reason this is in `start_link/1`, giving the table the lifetime of the caller (usually a supervisor), instead of in `init/1`?

Reply:
Oh good catch, no good reason. I'm moving it to `init/1`.